app/lib/utils/i18n.js

```diff
@@ -93,30 +93,172 @@ export const locales = [
   'zh-TW'
 ]
 
-// Defaine list of currencies that we will support.
+// Define list of currencies that we will support.
```

Comment:
If we are going to define a complete list like this, could/should we pull the list from the `country-data-lookup` package, which is already installed and imported into this library?

See https://github.com/Zooz/country-data

You can fetch a list of all currencies buy calling `currencies.all`